src/main/osd/osd.h

```diff
@@ -226,6 +226,15 @@ STATIC_ASSERT(OSD_WARNING_COUNT <= 32, osdwarnings_overflow);
 #define ESC_TEMP_ALARM_OFF INT8_MIN
 #define ESC_CURRENT_ALARM_OFF -1
 
+
+#ifndef CORE_TEMPERATURE_THRESHOLD
+#ifdef STM32H7
+#define CORE_TEMPERATURE_THRESHOLD 85
+#else
+#define CORE_TEMPERATURE_THRESHOLD 70 // a temperature above 70C should produce a warning, lockups have been reported above 80C.
```

Comment:
Part of the comment should be moved up before `#ifdef` with H7 specific value amended why the value is different from others.

I remember you were mentioning something like 70 degC on SPRH7X, which I think is unusually high. A bare Nucleo-H743ZI2 (LPFQ-144) running at 540MHz stays at 50 degC at room temp...

Reply:
Ok, will move comment.

Some background information:

There's quite a lot of thermal mass in a Nucleo 144 PCB to dissipate the heat,  conversely on a 27mm*27mm board there is nowhere for the heat to go except down the wires soldered to the PCB.  Couple that with the high-current draw of the H7 at only 400Mhz with power/voltage scaling set to it's current value and the thermal resistance of most 3.3v LDO regulators + a 5v switching regulator and you get a fairly warm PCB!

Ideally this should be a per-manufacturer per-board configurable read-only setting since only the manufacturer knows the limits of their design, but we're not quite ready for that yet.

Reply:
> Ideally this should be a per-manufacturer per-board configurable read-only setting since only the manufacturer knows the limits of their design, but we're not quite ready for that yet.

With unified targets we will be - manufacturer will be able to adjust the defaults for anything that is configurable.